qa/suites/mixed-system-tests/single/tasks/basic.yaml

```diff
@@ -2,6 +2,49 @@ tasks:
 - exec:
     client.0:
       - sudo yum install python3 -y
```

Comment:
@sunilkumarn417 python3 gets installed in all the machines from cm-ansible. we can remove it from here.